tests/taskcluster/bin/raptor_graph

```diff
@@ -51,11 +56,11 @@ var config = {
     params: {
       provisionerId: 'aws-provisioner-v1',
       workerType: 'gaia',
-      raptorRuns: 15,
+      raptorRuns: 30,
       raptorApps: 'settings',
       gaiaRev: process.env.GITHUB_HEAD_REV,
       raptorTimeout: 600000,
-      raptorBootSleep: 180,
+      headMinusOne: process.env.GITHUB_PULL_REQUEST === 1 ? false : true,
```

Comment:
Hrm, not sure if I'm crazy about the name here, but that's a nit and I suck at naming, so good for me! :+1: